Commands/rottentomatoes.py

```
@@ -1,107 +1,152 @@
+        hits = _search_rt(query, year=year)
+
+        if not hits:
+            self.send_privmsg(cmd.channel, "No results found.")
+            return
+        
+        first_result = hits[0]
+        formatted_result = _format(first_result)
+
+        self.send_privmsg(cmd.channel, formatted_result)
+
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)